[2025-10-11T06:10:01.488+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-11T06:10:01.535+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline_flow.load_ratings_to_gold scheduled__1998-12-27T00:00:00+00:00 [queued]>
[2025-10-11T06:10:01.548+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline_flow.load_ratings_to_gold scheduled__1998-12-27T00:00:00+00:00 [queued]>
[2025-10-11T06:10:01.549+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-10-11T06:10:01.568+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): load_ratings_to_gold> on 1998-12-27 00:00:00+00:00
[2025-10-11T06:10:01.583+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=10521) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-10-11T06:10:01.585+0000] {standard_task_runner.py:64} INFO - Started process 10527 to run task
[2025-10-11T06:10:01.583+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'pipeline_flow', 'load_ratings_to_gold', 'scheduled__1998-12-27T00:00:00+00:00', '--job-id', '828', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_dag.py', '--cfg-path', '/tmp/tmp0k02cbwq']
[2025-10-11T06:10:01.586+0000] {standard_task_runner.py:91} INFO - Job 828: Subtask load_ratings_to_gold
[2025-10-11T06:10:01.670+0000] {task_command.py:426} INFO - Running <TaskInstance: pipeline_flow.load_ratings_to_gold scheduled__1998-12-27T00:00:00+00:00 [running]> on host 321597a50810
[2025-10-11T06:10:01.858+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='oluwaseyiogundimu10@gmail.com' AIRFLOW_CTX_DAG_OWNER='seyi' AIRFLOW_CTX_DAG_ID='pipeline_flow' AIRFLOW_CTX_TASK_ID='load_ratings_to_gold' AIRFLOW_CTX_EXECUTION_DATE='1998-12-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__1998-12-27T00:00:00+00:00'
[2025-10-11T06:10:01.860+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-11T06:10:02.085+0000] {s3_client.py:30} INFO - âœ… Successfully initialized S3 client.
[2025-10-11T06:10:02.087+0000] {read_bucket.py:26} INFO - Reading ratings/1999-01.csv from movie-pipeline-silver bucket.
[2025-10-11T06:10:03.920+0000] {read_bucket.py:36} ERROR - Error reading ratings/1999-01.csv from movie-pipeline-silver: An error occurred (NoSuchKey) when calling the GetObject operation: The specified key does not exist.
[2025-10-11T06:10:03.922+0000] {load.py:113} WARNING - Partition ratings/1999-01.csv not found. Skipping.
[2025-10-11T06:10:03.927+0000] {load.py:120} WARNING - No ratings data found in selected partitions. Skipping load.
[2025-10-11T06:10:03.931+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-10-11T06:10:03.932+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-11T06:10:03.957+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=pipeline_flow, task_id=load_ratings_to_gold, run_id=scheduled__1998-12-27T00:00:00+00:00, execution_date=19981227T000000, start_date=20251011T061001, end_date=20251011T061003
[2025-10-11T06:10:04.013+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-10-11T06:10:04.059+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-11T06:10:04.063+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
